controllers/antreainstall_controller.go

```diff
@@ -421,6 +420,21 @@ func deleteExistingPods(c client.Client, component string) error {
 func updateNetworkStatus(c cnoclient.Client, clusterConfig *configv1.Network, defaultMTU int) error {
 	status := configutil.BuildNetworkStatus(clusterConfig, defaultMTU)
 	clusterConfig.Status = *status
+
+	// Ensure networkDiagnostics has valid empty values to avoid validation errors, Openshift 4.15+
```

Comment:
I am aware this has been mentioned in the commit message, but can you share some details concerning these validation errors?

In which logs are they seen?

Reply:
Previously during apply the Network config crd didn't care about nil values for the placement values, and this is no longer the case, hence the need to set non nil values.

Logs for this is seen in the antre-operator pod it self due to the validation webhook failing.